backend/api/Program.cs

```diff
@@ -28,9 +28,18 @@ public static void Main(string[] args)
         public static IWebHostBuilder CreateWebHostBuilder(string[] args)
         {
             DotNetEnv.Env.Load();
+            var env = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT");
```

Comment:
To support running the api locally it required changes to support different ports.